camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -536,8 +536,8 @@ export default class Tasklist extends Mixins(TaskListMixin) {
     }
   }
 
-  async addGroup () {
-    await CamundaRest.createTaskGroupByID(
+  addGroup () {
```

Comment:
Why removed async/await after the addGroup?

Reply:
because we don't need that. we need to run the function asynchronous

Reply:
and also we are using the promise . we wrote the rest of the function inside the resolve state